doc/source/whatsnew/v2.0.2.rst

```diff
@@ -17,6 +17,7 @@ Fixed regressions
 - Fixed regression in :func:`merge` on Windows when dtype is ``np.intc`` (:issue:`52451`)
 - Fixed regression in :func:`read_sql` dropping columns with duplicated column names (:issue:`53117`)
 - Fixed regression in :meth:`DataFrame.loc` losing :class:`MultiIndex` name when enlarging object (:issue:`53053`)
+- Fixed regression in :meth:`DataFrame.loc` rows are not inserted into DataFrame from lists of one element (:issue:`52825`)
```

Comment:
the wording here is weird.  maybe "Fixed regression in DataFrame.loc when the dataframe is empty and the item being set is a single-element list"

Reply:
Done